vaadin-dashboard-flow-parent/vaadin-dashboard-flow/src/main/java/com/vaadin/flow/component/dashboard/DashboardVariant.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2000-2025 Vaadin Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package com.vaadin.flow.component.dashboard;
+
+import com.vaadin.flow.component.shared.ThemeVariant;
+
+/**
+ * The set of theme variants applicable to the {@code vaadin-dashboard} and
+ * {@code vaadin-dashboard-layout} components.
+ */
+public enum DashboardVariant implements ThemeVariant {
+    
+    LUMO_SHADED_BACKGROUND("shaded-background"),
+    LUMO_ELEVATED_WIDGETS("elvated-widgets"),
```

Comment:
Looks like a type to me:
```suggestion
    LUMO_ELEVATED_WIDGETS("elevated-widgets"),
```

Reply:
This is now fixed